M2/Macaulay2/packages/Text.m2

```diff
@@ -6,8 +6,8 @@ newPackage("Text",
      )
 
 exportFrom_Core {
-     "ANCHOR", "BLOCKQUOTE", "BODY", "BOLD", "BR", "CDATA", "CODE", "COMMENT", "DD", "DIV", "DL", "DT", "EM", "ExampleItem", "HEAD", "HEADER1",
-     "HEADER2", "HEADER3", "HEADER4", "HEADER5", "HEADER6", "HR", "HREF", "HTML", "Hypertext", "HypertextContainer", "HypertextParagraph", "IMG", "ITALIC",
+     "ANCHOR", "BLOCKQUOTE", "BODY", "BOLD", "BF", "BR", "CDATA", "CODE", "COMMENT", "DD", "DIV", "DL", "DT", "EM", "ExampleItem", "HEAD", "HEADER1",
+     "HEADER2", "HEADER3", "HEADER4", "HEADER5", "HEADER6", "HR", "HREF", "HTML", "Hypertext", "HypertextContainer", "HypertextParagraph", "IMG", "ITALIC", "IT",
```

Comment:
remove BF and IT

Reply:
I think remembering `@BF "..."@` and `@IT "..."@` is much easier for users who know latex, but the reason I mentioned above I mentioned why I added them.

Reply:
> I think remembering `@BF "..."@` and `@IT "..."@` is much easier for users who know latex, but I mentioned above the reason why I added them.